### PR TITLE
DOCS-2482: Quickstart detect people with a webcam

### PR DESCRIPTION
**Pull request by sguequierre**
* Make a quickstart for detecting people with a webcam

[can maybe take better/less cluttered detections photo at office?]

### CODE REVIEW

File: docs/get-started/quickstarts/detect-people.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+description: "Use Viam's machine learning capabilities to deploy a person detector ML model to your machine."
+images: ["/platform/ml.svg"]
+imageAlt: "Machine Learning"
+---
+
+Follow this guide to detect people with a webcam on the Viam platform, using a [pre trained ML model from the registry](https://app.viam.com/ml-model/ml-models-scuttle/people).
+
+## Requirements
+
+- A computer: a development machine like your laptop and optionally a single-board computer like the Raspberry Pi
```
Comment (andf-viam) [inline docs/get-started/quickstarts/detect-people.md:17]: Nit: phrasing:

```suggestion
- A computer: a development machine such as your laptop and optionally a single-board computer such as the Raspberry Pi
```

File: docs/get-started/quickstarts/detect-people.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+- A webcam: this could be the webcam on your laptop or any other webcam you can connect to your computer
```
Comment (andf-viam) [inline docs/get-started/quickstarts/detect-people.md:18]: Opt: Non-webcam cameras could work here as well, but the suggestion of a webcam to start is definitely the way to go. Perhaps something like:

```suggestion
- A camera: the webcam on your development machine or a camera device connected to your single-board computer
```

However, if we want to orient this whole quickstart towards webcams specifically, I'm ok with that, disregard this comment!

File: docs/get-started/quickstarts/detect-people.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+- An account on the [Viam app](https://app.viam.com)
+
+First, follow the [instructions to create a new machine](/cloud/machines/#add-a-new-machine) representing your computer.
```
Comment (andf-viam) [inline docs/get-started/quickstarts/detect-people.md:21]: For these quickstarts, I think we want everything needed to complete the task presented on this page, ideally in the same format (expanders in this case), rather than linking off-page to other resources. What do you think of:

1. Convert /the `cloud/machines/#add-a-new-machine` instructions to expander, and repeat that content here (but absolutely use the same image file!)

2. Convert "Follow the setup instructions" to a second expander, saying verbatim what you already have here (App UI is separate from docs, no way around sending users there.)

3. Delete the TOC for this page that is lines 26 - 28: This section forecasts the next three steps, but the fact that they are linked (to their respective reference content) strongly suggests to the reader that these are in fact prerequisites that they must complete first, before then opening the first expander below.

Altogether, this looks to me like something resembling:

```
{{%expand "Step 1: Create a new machine" %}}
# Dupe /cloud/machines/#add-a-new-machine - it's two sentences and an image you can reuse
{{%expand "Step 2: Install `viam-server`" %}}
# Verbatim copy to what you already have
{{%expand "Step 3: Configure your webcam" %}}
{{%expand "Step 4: Deploy a person detection model" %}}
{{%expand "Step 5: Configure your detector" %}}
{{%expand "Step 6: Test person detection" %}}
```

Expanders for steps 4 -6 can then contain the link to the related reference entry for each that you previously used as the TOC: lines 26-28. Within the expander, the user may choose to click through to more info about this current expander topic (your link), or skip past and open the next one: the IA is clearer.

File: docs/get-started/quickstarts/detect-people.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+Follow the {{< glossary_tooltip term_id="setup" text="setup instructions" >}} that appear on your new machine's **CONFIGURE** page to install `viam-server` on your computer and connect it to the Viam app.
```
Comment (andf-viam) [inline docs/get-started/quickstarts/detect-people.md:22]: Per comment above, suggest converting to expander.

File: docs/get-started/quickstarts/detect-people.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+You need to configure the following {{< glossary_tooltip term_id="resource" text="resources" >}} to detect people with a webcam:
+
+1. A [`webcam` camera](/components/camera/webcam/)
+2. An [`tflite_cpu` ML model service](/services/ml/deploy/tflite_cpu/)
+3. An [`mlmodel` computer vision service](/services/vision/mlmodel/)
```
Comment (andf-viam) [inline docs/get-started/quickstarts/detect-people.md:28]: Per comment above, suggest deleting these lines. Feel free to re-use these links within their respective expander, though!

File: docs/get-started/quickstarts/detect-people.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+Follow these instructions to configure your machine and test detections:
+
+{{%expand "Step 1: Configure your webcam" %}}
+
+Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
+Select the `camera` type, then select the `webcam` model.
+Enter a name or use the suggested name for your camera and click **Create**.
+
+Click the **Save** button in the top right corner of the page to save your config.
```
Comment (andf-viam) [inline docs/get-started/quickstarts/detect-people.md:39]: Nit: full word unless referring to UI element or code verbatim. Unpopular opinion though, feel free to overrule me if you strongly disagree 👍 

```suggestion
Click the **Save** button in the top right corner of the page to save your configuration.
```

File: docs/get-started/quickstarts/detect-people.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+Click the **+** icon next to your machine part in the left-hand menu and select **Service**.
+Select the `ML model` type, then select the `TFLite CPU` model.
+Enter a name or use the suggested name for your service and click **Create**.
+
+To deploy a person detector model from the registry, select **Deploy model on machine** for the **Deployment** field in the resulting vision service configuration pane.
+
+Click on **Select models** to open a dropdown with all of the ML models available to you.
+Select the **people** model by **ml-models-scuttle** to deploy an object detection TFLite model that's pre trained with person detection.
+
+Click the **Save** button in the top right corner of the page to save your config.
```
Comment (andf-viam) [inline docs/get-started/quickstarts/detect-people.md:58]:
```suggestion
Click the **Save** button in the top right corner of the page to save your configuration.
```

File: docs/services/ml/deploy/tflite_cpu.md [hunk 8]
```diff
@@ -38,7 +39,7 @@ You can choose to configure your service with an existing model on the machine o
 {{< tabs >}}
 {{% tab name="Deploy Model on Machine" %}}
 
-1. To configure your service and deploy a model onto your machine, select **Deploy model on machine** for the **Deployment** field in the resulting vision service configuration pane,.
+1. To configure your service and deploy a model onto your machine, select **Deploy model on machine** for the **Deployment** field in the resulting vision service configuration pane.
```
Comment (andf-viam) [inline docs/services/ml/deploy/tflite_cpu.md:42]: Thanks! Nice catch.

File: docs/get-started/quickstarts/detect-people.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
+Select the `camera` type, then select the `webcam` model.
+Enter a name or use the suggested name for your camera and click **Create**.
+
+Click the **Save** button in the top right corner of the page to save your config.
+Go to the **CONTROL** tab and expand the camera's remote control card to test the camera stream.
+If you don't see an image stream, you need to [configure the `video_path` attribute](/components/camera/webcam/#using-video_path).
+
+For more detailed information, including optional attribute configuration, see the [`webcam` docs](/components/camera/webcam/).
```
Comment (andf-viam) [inline docs/get-started/quickstarts/detect-people.md:43]: If you like, the lines preceeding this line are all stepwise instructions: they might look good as bullet points within this expander. Maybe something like this:

```
- Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com/).
- Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
- Select the `camera` type, then select the `webcam` model.
- Enter a name or use the suggested name for your camera and click **Create**.
- Click the **Save** button in the top right corner of the page to save your config.
- Go to the **CONTROL** tab and expand the camera's remote control card to test the camera stream.
   If you don't see an image stream, you need to [configure the `video_path` attribute](/components/camera/webcam/#using-video_path).

For more detailed information, including optional attribute configuration, see the [`webcam` docs](/components/camera/webcam/).
```

If you like this approach and use it for this expander, suggest expanding this practices to the other ... expanders. If you don't like this approach, and want to keep it conversational, consider using segue words between some sentence so they are not all imperative directives: "Next ... " "First", etc. Minor!

File: docs/get-started/quickstarts/detect-people.md [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+
+{{% /expand%}}
+{{%expand "Step 2: Deploy a person detection model" %}}
+
+Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+Click the **+** icon next to your machine part in the left-hand menu and select **Service**.
+Select the `ML model` type, then select the `TFLite CPU` model.
+Enter a name or use the suggested name for your service and click **Create**.
+
+To deploy a person detector model from the registry, select **Deploy model on machine** for the **Deployment** field in the resulting vision service configuration pane.
```
Comment (andf-viam) [inline docs/get-started/quickstarts/detect-people.md:53]: Nit: the user is still looking at the left nav in app at this point, while reading through this sentence, to find `in the resulting vision service configuration pane.` at the end telling them where to shift their eyes. Instead, start with this instruction to help them find it first, before then explaining what to do. Something like:

```suggestion
In the resulting ML model service configuration pane, ensure that **Deploy model on machine** is selected for the **Deployment** field.
```

It's also set to this value by default, so using "ensure" instead, feel free to tweak!

File: docs/get-started/quickstarts/detect-people.md [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+
+Click on **Select models** to open a dropdown with all of the ML models available to you.
+Select the **people** model by **ml-models-scuttle** to deploy an object detection TFLite model that's pre trained with person detection.
```
Comment (andf-viam) [inline docs/get-started/quickstarts/detect-people.md:56]: Supernit: optionally include the exact string we expect to see, since we are otherwise presenting the two parts in opposite order that they appear (for left-right language readers): `ml-models-scuttle:people`.

Maybe:

```suggestion
Select the `ml-models-scuttle:people` model (the **people** model by **ml-models-scuttle**) to deploy an object detection TFLite model that has been trained to be able to detect a person.
```

File: docs/get-started/quickstarts/detect-people.md [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+
+{{% /expand%}}
+{{%expand "Step 3: Configure your detector" %}}
+
+Navigate to the **CONFIGURE** tab of your machine's page in the [Viam app](https://app.viam.com).
+Click the **+** icon next to your machine part in the left-hand menu and select **Service**.
+Select the `vision` type, then select the `ML model` model.
+Enter a name or use the suggested name for your service and click **Create**.
+
+Select the ML model service your person detector model is deployed on from the **ML Model** dropdown.
```
Comment (andf-viam) [inline docs/get-started/quickstarts/detect-people.md:70]: Opt: reference when we created this earlier, maybe:

```suggestion
Select the ML model service your person detector model is deployed on (which you created in step X) from the **ML Model** dropdown.
```

File: docs/get-started/quickstarts/detect-people.md [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+
+Click the **Save** button in the top right corner of the page.
+
+For more detailed information, including optional attribute configuration, see the [`mlmodel` docs](/services/ml/vision/mlmodel/).
+
+{{% /expand%}}
+
+{{%expand "Step 4: Control person detection" %}}
+
+Now, test your person detection out with the computer vision service's remote control card.
```
Comment (andf-viam) [inline docs/get-started/quickstarts/detect-people.md:80]:
```suggestion
Now, test your person detection with the computer vision service's remote control card.
```

File: docs/get-started/quickstarts/detect-people.md [hunk 14]
```diff
@@ -0,0 +1,108 @@
...
+{{% /expand%}}
+
+## Next steps
+
+You can continue to test your [`mlmodel` detector](/services/vision/mlmodel/#test-your-detector-or-classifier) with [existing images in the Viam app](/services/vision/mlmodel/#existing-images-in-the-cloud) or [existing images on a computer](/services/vision/mlmodel/#existing-images-on-your-machine).
+
+You can use one of [Viam's SDKs](/sdks/) to write a script that captures detections and triggers an alarm when a person is detected, or even sends you photos of the person detected.
+See [A Person Detection Security Robot That Sends You Photos](/tutorials/projects/send-security-photo/#use-the-viam-python-sdk-to-control-your-security-robot) for an example of this.
+
+You can also try using the Viam platform to [train your own detector or classifier](/use-cases/deploy-ml/).
```
Comment (andf-viam) [inline docs/get-started/quickstarts/detect-people.md:100]:
```suggestion
You can also use the Viam platform to [train your own detector or classifier](/use-cases/deploy-ml/).
```

**Review by andf-viam — COMMENTED** [review]
This is looking great! I've left some smallish comments on phrasing and the like, but I think we also want to simply the presentation of introductory content. I left a comment exploring expanding the use of ... expanders and removing some of the intro content accordingly. 

I think the image could work, but would be better if it showcased 2+ person matches and fewer false positives (casual reader go: "this model skilled at missing refrigerators"). If you do use an image with many false positives, consider adding a sentence underneath referencing the confidence scores, explaining. Very opt!

Great work!

File: docs/get-started/quickstarts/detect-people.md (hunk 2 above)
Comment (sguequierre) [reply docs/get-started/quickstarts/detect-people.md:18]: I was thinking webcam specifically bc it was was labeled as so [on ticket ](https://viam.atlassian.net/jira/software/c/projects/DOCS/boards/31?quickFilter=40&selectedIssue=DOCS-2481)

File: docs/get-started/quickstarts/detect-people.md [hunk 15]
```diff
@@ -0,0 +1,114 @@
...
+Navigate to the **CONFIGURE** tab of your machine's page in the [Viam app](https://app.viam.com).
+Click the **+** icon next to your machine part in the left-hand menu and select **Service**.
+Select the `vision` type, then select the `ML model` model.
+Enter a name or use the suggested name for your service and click **Create**.
+
+Select the ML model service your person detector model is deployed on (which you created in step 4) from the **ML Model** dropdown.
+
+Click the **Save** button in the top right corner of the page.
+
+For more detailed information, including optional attribute configuration, see the [`mlmodel` docs](/services/ml/vision/mlmodel/).
```
Comment (sguequierre) [inline docs/get-started/quickstarts/detect-people.md:80]:
```suggestion
For more detailed information, including optional attribute configuration, see the [`mlmodel` docs](/services/vision/mlmodel/).
```

File: docs/get-started/quickstarts/detect-people.md (hunk 2 above)
Comment (andf-viam) [reply docs/get-started/quickstarts/detect-people.md:18]: Right on, works for me!

File: assets/get-started/quickstarts/detect-people.md
Comment (andf-viam) [inline assets/get-started/quickstarts/detect-people.md:1]: I think this dupe `md` file can be deleted.

**Review by andf-viam — APPROVED** [review]
Awesome job, this LGTM mod one fix:
- You have an extra copy of your quickstart in `assets` that needs to be deleted.

Thanks, this looks great! You could even bump up your pic to above the steps if you like: really showcases the goal of the quickstart. Optional!

File: docs/get-started/quickstarts/detect-people.md [hunk 16]
```diff
@@ -0,0 +1,125 @@
+---
+title: "Detect People with a Webcam"
+linkTitle: "Detect people with a webcam"
+weight: 50
+type: "docs"
+tags: ["vision", "data", "services", "quickstart", "ml", "camera", "webcam"]
+no_list: true
+description: "Use Viam's machine learning capabilities to deploy a person detector ML model to your machine."
+images: ["/platform/ml.svg"]
```
Comment (JessamyT) [inline docs/get-started/quickstarts/detect-people.md:9]: Sorry for drive-by comment; am looking at this for reference of another quickstart. Maybe a camera icon would be better since this plant watering one doesn't really fit?

Comment (sguequierre) [inline docs/get-started/quickstarts/detect-people.md:9]:
```suggestion
images: ["/icons/components/camera.svg"]
```
